### PR TITLE
Created new theme: Dark pink

### PR DESCRIPTION
**Pull request by miicat**
Created Dark pink theme based on Alternative Dark theme

### CODE REVIEW

File: p/themes/Dark-pink/icons/add-white.svg [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   width="16"
+   height="16"
+   version="1.1"
+   id="svg4"
+   sodipodi:docname="add-white.svg"
+   inkscape:version="1.1.2 (0a00cf5339, 2022-02-04, custom)"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
```
Comment (Alkarex) [inline p/themes/Dark-pink/icons/add-white.svg]: @math-GH has a nice method to optimize SVGs, including removing such proprietary elements. Do you have it documented anywhere @math-GH ?

**Comment by Alkarex** [comment]
Hello @miicat and welcome!
Looks nice, thanks :-)

Quick review: the text for the headers of the logs lacks contrast:
image

**Comment by miicat** [comment]
Hi @Alkarex,
Thanks :3

Good testing. I will fix that and optimize the SVGs once I know how to

**Comment by math-GH** [comment]
I would prefer to use filter in CSS instead of new icon files

Could you please try this?
```
.icon, 
.header > .item.title .logo {
  filter: invert(89%) sepia(100%) saturate(214%) hue-rotate(287deg) brightness(104%) contrast(102%)
}
```

This page helps to get a good starting point to create filter parameters:
https://codepen.io/sosuke/pen/Pjoqqp

The tool to optimize SVG files: https://jakearchibald.github.io/svgomg/

**Comment by miicat** [comment]
I fixed the bug with the log's navigation buttons and optimized the SVGs

I looked into the `filter`, but figured it wouldn't be that easy as some images have different colors (and also, those may change in the future)

Let me know, if you find anything else that needs to be fixed :3

**Comment by math-GH** [comment]
Icons:
I would prefer to have a transparent background instead of a black background.
(f.e. `p/themes/Dark-pink/icons/favicon.svg` `p/themes/Dark-pink/icons/favicon-96-padding.png` `p/themes/Dark-pink/icons/favicon-256.png` etc)


`p/themes/Dark-pink/icons/FreshRSS-logo.svg`: I would prefer to use CSS filter. Filter are used in all other themes to color the FreshRSS icon.

In general I want to point that most themes use CSS filters to color the icons. It reduces the effort to maintain the themes. For example: see "Flat design" theme. The grey icons are made white with CSS filters, except the star and red envelop icon:
```
#toggle-unread .icon, #toggle-starred .icon {
filter: none;
}
```

From point of theme maintaining, I would suggest:
- use CSS filters instead of new icons
- copy dark.css into this new theme
- add pinkdark.css and add here only the needed lines to color it.

File: p/themes/Dark-pink/metadata.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Dark pink",
+  "author": "Miicat_47",
+  "description": "Nice dawk pink theme. >_< UwU",
+  "version": 0.1,
+  "files": ["_template.css", "pinkdark.css"]
```
Comment (Alkarex) [inline p/themes/Dark-pink/metadata.json]:
```suggestion
  "files": ["_template.css", "../Dark/dark.css", "pinkdark.css"]
```

@math-GH Alternative suggestion to copying `dark.css`

Comment (math-GH) [reply p/themes/Dark-pink/metadata.json]: thanks. I thought about this too, but did not checked if it works. Great.

Comment (miicat) [reply p/themes/Dark-pink/metadata.json]: Nice, did that

**Comment by math-GH** [comment]
Somehow I do not have a pink FreshRSS logo.
And the dropdown arrow needs some other color (please check: in theme "dark" there is a lighter border around the dropdown that is removed/not visible in this pink theme)

grafik

**Comment by miicat** [comment]
@math-GH I'm not ready yet, I will let you know, when you can take another look

**Comment by miicat** [comment]
@math-GH You can take another look now :3

File: p/themes/Dark-pink/pinkdark.css [hunk 3]
```diff
@@ -0,0 +1,125 @@
+.btn:hover {
+	background: unset !important;
```
Comment (Alkarex) [inline p/themes/Dark-pink/pinkdark.css]: Is is possible to reduce the use of the many `!important` or are they actually needed?

Comment (miicat) [reply p/themes/Dark-pink/pinkdark.css]: I think they are actually needed, I can test without them tho

Comment (miicat) [reply p/themes/Dark-pink/pinkdark.css]: Yep, `!important` were unnecessary, removed them

**Comment by Alkarex** [comment]
Good for you @math-GH ?

**Comment by Alkarex** [comment]
I have tested a bit on my side and it looks fine. Let's merge to potentially gather a bit more feedback :-)

**Comment by miicat** [comment]
I have been using the theme daily and haven't found any new bugs

**Pull request merged**